api/src/main/kotlin/com/gmtkgamejam/routing/PostRoutes.kt

```diff
@@ -210,83 +189,3 @@ fun Application.configurePostRouting() {
         }
     }
 }
-
-fun getFilterFromParameters(params: Parameters): List<Bson> {
```

Comment:
I cannot tell you how thrilled I am to have deleted all of this

Reply:
Even more thrilled than I am to see it go :grin: